tests/network-tests/src/flows/contentDirectory/creatingVideo.ts

```diff
@@ -42,8 +43,65 @@ export function createVideoReferencingChannelFixture(api: QueryNodeApi): CreateV
   return new CreateVideoFixture(api, videoEntity)
 }
 
+function assertVideoMatchQueriedResult(queriedVideo: any, video: VideoEntity) {
+  assert(queriedVideo.title === video.title, 'Should be equal')
+  assert(queriedVideo.description === video.description, 'Should be equal')
+  assert(queriedVideo.duration === video.duration, 'Should be equal')
+  assert(queriedVideo.thumbnailUrl === video.thumbnailUrl, 'Should be equal')
+  assert(queriedVideo.isExplicit === video.isExplicit, 'Should be equal')
+  assert(queriedVideo.isPublic === video.isPublic, 'Should be equal')
+}
+
 export default async function createVideo(api: QueryNodeApi) {
-  const createVideoHappyCaseFixture = createVideoReferencingChannelFixture(api)
+  const channelTitle = 'New channel example'
+  const createVideoHappyCaseFixture = createVideoReferencingChannelFixture(api, channelTitle)
 
   await createVideoHappyCaseFixture.runner(false)
+
+  // Temporary solution (wait 2 minutes)
+  await Utils.wait(120000)
+
+  // Perform number of full text searches on Channel title, that is a slight variation on title that one expects would return the video.
+  let channelFullTextSearchResult = await api.performFullTextSearchOnChannelTitle('video')
+
+  assert(channelFullTextSearchResult.data.search.length === 1, 'Should contain exactly one entry')
+
+  // Both channel and video title starts with `Example`
+  channelFullTextSearchResult = await api.performFullTextSearchOnChannelTitle('Example')
```

Comment:
the name of the function is misleading, if it is searching both channels and videos?

Reply:
Currently full text search on  Channel/Video title returns both Channel/Video and its respective Channel/Video entities, if their full-text search fields (handle, etc.) can be accessed by the the same full text search query.
Have a look at second point here https://github.com/Joystream/joystream/issues/1552
`Query the query node with a number of full text searches on the channel title, that all should be slight variations on title that one expects would return the video.`

I guess, it`s because they have db relation.